rflint/rflint.py

```diff
@@ -111,6 +111,12 @@ def run(self, args):
             else:
                 self._process_file(filename)
 
+        if self.args.summary:
+            print("----------------------------------------------------------------------",
+                  f"Found {sum([x for x in self.counts.values()])} issues: "
```

Comment:
sum(self.counts.values()) should work too

Reply:
You're right, I was changing this line several times and end up with this one. Will definitely change it to your idea.